src/canvas/.env.example

```diff
@@ -0,0 +1,18 @@
+CSV_INPUT_BASE_PATH=/edfi/Ed-Fi-X-Fizz/canvas/output/extracted-data
+CSV_OUTPUT_PATH=/edfi/Ed-Fi-X-Fizz/src/canvas/output/generated-data
+
+data extractor py
+CANVAS_BASE_URL=http://fizz-canvas.centralus.cloudapp.azure.com
+ACCESS_TOKEN=gd040lThLXSWTFJXNOd3cOCRWziYok7u0NasunDYmnForHYXm72e92buTD2zECxT
```

Comment:
Let's change the example to have a less-realistic URL and access token. And you'll need to invalidate that access token now :-D.

Reply:
Yeah... My bad... I rebased and removed this commit.